src/components/ExpandableContainer/index.tsx

```diff
@@ -7,13 +7,15 @@ import {
 
 import useBooleanState from '../../hooks/useBooleanState';
 import Container, { Props as ContainerProps } from '../Container';
+import QuickActionButton from '../QuickActionButton';
 
 import styles from './styles.css';
 
 export interface Props extends Omit<ContainerProps, 'containerElementProps'>{
     defaultVisibility?: boolean;
     // NOTE: Mount will mount the child even if its not shown
     alwaysMountContent?: boolean;
+    expandOnButtonClick?: boolean;
```

Comment:
Lets use better prop name
```suggestion
    expansionTriggerArea?: 'header' | 'arrow'
```